openquake/hazardlib/calc/filters.py

```diff
@@ -276,8 +276,9 @@ def __init__(self, sitecol, integration_distance, use_rtree=True):
         self.use_rtree = use_rtree and rtree and (
             integration_distance and sitecol is not None and
             sitecol.at_sea_level())
-        if self.use_rtree:
+        if sitecol is not None:
```

Comment:
As it's written here, it is not so intuitive. The fact that `if self.use_rtree` implies that also sitecol is not None does not look obvious at a first sight.